src/Commands/AliasesCommand.php

```diff
@@ -2,38 +2,266 @@
 
 namespace Pantheon\Terminus\Commands;
 
-use Pantheon\Terminus\Helpers\LocalMachineHelper;
+use Pantheon\Terminus\Commands\TerminusCommand;
 
-class AliasesCommand extends TerminusCommand
+use Pantheon\Terminus\Collections\Sites;
+use Pantheon\Terminus\Site\SiteAwareInterface;
+use Pantheon\Terminus\Site\SiteAwareTrait;
+
+use Pantheon\Terminus\Helpers\AliasEmitters\AliasCollection;
+use Pantheon\Terminus\Helpers\AliasEmitters\AliasData;
+use Pantheon\Terminus\Helpers\AliasEmitters\AliasesDrushRcEmitter;
+use Pantheon\Terminus\Helpers\AliasEmitters\PrintingEmitter;
+use Pantheon\Terminus\Helpers\AliasEmitters\DrushSitesYmlEmitter;
+
+/**
+ * Generate lots of aliases
+ */
+class AliasesCommand extends TerminusCommand implements SiteAwareInterface
 {
+    use SiteAwareTrait;
+
     /**
      * Generates Pantheon Drush aliases for sites on which the currently logged-in user is on the team.
      *
-     * @authorize
+     * @authenticated
      *
      * @command aliases
-     * @aliases drush:aliases
+     * @aliases alpha:aliases
```

Comment:
We should note this removal and addition in the CHANGELOG.md.

Reply:
The removal was accidental. I'll put it back.